### DISCUSSION TITLE
new empty workspace?

### DISCUSSION THREAD

**Discussion by jjgalvez** [toeverything/AFFiNE#4018] [Q&A] [answered]
Is there a way to create a new empty workspace? I find I have to delete all the demo content when I create a new workspace.

**Comment by himself65** [comment]
@houjoe0829

**Comment by houjoe0829** [comment, marked as answer]
Sorry, there is currently no way to create a completely empty Workspace, but we will support batch operations in the future and can follow up on https://github.com/toeverything/AFFiNE/issues/2305 .

**Comment by jjgalvez** [comment]
@houjoe0829  Thank you I'll follow the batch operations issue you linked
